ProjetDevopsPanda/src/main/java/BreackfastClub/ProjetDevopsPanda/Dataframe.java

```diff
@@ -100,25 +133,152 @@ public void addLine(Object[] line) {
 		}
 	}
 	
+	/**
+	 * @param indice is the position of the column you want to change
+	 * @param col is the content of the column you want to put in the dataframe
+	 * @throws DataframeNullException 
+	 */
+	public void setCol(Object indice, Object[] col) throws DataframeNullException {
```

Comment:
The function setCol doesn't work as intended. The number of arguments is not verified in the function and has to be.